indigo/bulk_creator.py

```diff
@@ -466,6 +470,21 @@ def link_amendment(self, work, info):
             amendment.date = date
             amendment.save()
 
+    def link_taxonomy(self, work, info):
+        topics = info.get('taxonomy').split(', ')
```

Comment:
This will break when a user enters "foo,bar, boom,   baz" accidentally. I suggest splitting on either a comma and then stripping whitespace, or splitting on whitespace and then stripping commas.

Reply:
```suggestion
        topics = [x.strip(",") for x in info.get('taxonomy').split()]
```